runtime/resolvers/api.go

```diff
@@ -6,8 +6,11 @@ import (
 
 	"github.com/mitchellh/mapstructure"
 	"github.com/rilldata/rill/runtime"
+	"go.opentelemetry.io/otel"
 )
 
+var tracer = otel.Tracer("github.com/rilldata/rill/runtime/resolvers")
```

Comment:
nit: maybe put in a common file called `resolvers.go` instead of `api.go` in the package?